src/nuget/Microsoft.DotNet.BuildTools.Run.nuspec

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata minClientVersion="2.8.1">
+    <id>Microsoft.DotNet.BuildTools.Run</id>
+    <version>1.0.0-prerelease</version>
+    <title>Microsoft DotNet Run Tool</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <projectUrl>http://go.microsoft.com/fwlink/?LinkId=518631</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <summary>.NET Core Framework Run Tool</summary>
+    <description>This package provides build-time support for the .NET Core Framework libraries.  You should not need to reference this package from your project.</description>
+    <copyright>Copyright © Microsoft Corporation</copyright>
+    <tags></tags>
+    <dependencies>
+    <!--
+      Do not add any dependencies to this package as we want to install it versionless
+      (i.e. -ExcludeVersion) and we don't want the dependencies to be installed that way
+    -->
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="run\run.exe" target="tools\netcoreapp1.0" />
```

Comment:
For the CLI version can we use a shared CLI-bootstrapper script on an HTTPS endpoint (still a bit queasy about this but how else can we share it) that is downloaded as part of the run bootstrapper?  This way any tool that requires a CLI can use this script and the version is consistent.  Of course it does mean that as the script rolls the version forward all tools that use it will also roll forward.  So the process would look something like the following.

run.bat checked into Foo repo, downloads CLI bootstrapper script and executes
run.bat restores version of run package and executes "real" run.bat script in the package which does the actual work